forgerock-authenticator/lib/forgerock_authenticator.dart

```diff
@@ -38,6 +38,10 @@ class ForgerockAuthenticator {
   static const PushRegistrationException = 'PUSH_REGISTRATION_EXCEPTION';
   static const PolicyViolationException = 'POLICY_VIOLATION_EXCEPTION';
 
+  static const PUSH_URI = 'pushauth';
+  static const OATH_URI = 'otpauth';
+  static const MFAUTH_URI = 'mfauth';
```

Comment:
Why are we adding those here? I don't see them being used anywhere?

Reply:
Because we need to compare the URI schemes and this is the main class for the Plugin. We are using in the official Authenticator app in the other project.